docs/Breaking-Changes.md

```diff
@@ -4,7 +4,7 @@
 
 ## HEAD ##
 
-_none_
+  * Add the exception throw when the negative is converted to unsigned integer for template `convert<type>`.
```

Comment:
Reword:

"Throws an exception when trying to parse a negative number as an unsigned integer."

Reply:
ok，done